mongo/writeconcern/writeconcern.go

```diff
@@ -224,6 +391,8 @@ func (wc *WriteConcern) WithOptions(options ...Option) *WriteConcern {
 }
 
 // AckWrite returns true if a write concern represents an acknowledged write
+//
+// Deprecated: AckWrite will not be supported in Go Driver 2.0.
```

Comment:
Should we include a migration suggestion?
```suggestion
// Deprecated: AckWrite will not be supported in Go Driver 2.0. Call "Acknowledge" method directly. 
// For example:
//
//        wc.Acknowledge()
```

Reply:
Good idea. I will actually just replace the text with an alternative recommendation to match the format in other deprecation notices.